docs: neon local and local connect (#3840)
neondatabase/website: content/guides/neon-local.md

let sslConfig;

if (process.env.NODE_ENV === 'development') {
  sslConfig = { rejectUnauthorized: false };
}

export const drizzleClient = drizzle({
  connection: {
    connectionString,
    ssl: sslConfig,
  },
});
```

For **Prisma** with the standard Postgres driver you again only need to set the database URL appropriately for your environment. Prisma will handle the rest:

```typescript
import { PrismaClient } from './generated/prisma/client.js';

const prismaClient = new PrismaClient({
  datasources: {
    db: {
      url: process.env.DATABASE_URL || 'postgres://neon:npg@localhost:5432/<database_name>',
    },
  },
});
```

- In any case, use the local Neon Local connection string for development and your production Neon connection string in deployed environments.
- Conditionally disabling SSL for local development ensures compatibility with the Neon Local proxy, while production connections remain secure.

</TabItem>
</Tabs>

<Admonition type="note" title="Other languages and drivers">
The complex `neonConfig` setup is **exclusive to the `@neondatabase/serverless` driver** and its wrappers (Drizzle, Prisma adapter) in Node.js environments.

> For applications written in other languages (like Python, Go, Ruby, Java, etc.) that use standard PostgreSQL drivers, the process is much simpler. It follows the same pattern as the `node-postgres (pg)` example:

Reviewer: ```suggestion
For applications written in other languages (like Python, Go, Ruby, Java, etc.) that use standard PostgreSQL drivers, you can follow the same pattern as the `node-postgres (pg)` example:
```
